fixed overlay opacity issue in the user menu component

RaghavendranC-Eaton opened the pull request:
<!-- If this pull request fixes an Issue, link it below. If not, you can remove the line below -->

Fixes #156 .

<!-- Include a bulleted list summarizing the main changes you have made in this PR -->

#### Changes proposed in this Pull Request:

- update the overlay style in the light and dark theme to resolved this issue
-
-

<!-- Include screenshots if they will help illustrate the changes in this PR -->

#### Screenshots / Screen Recording (if applicable)

- 
![image](https://github.com/etn-ccis/blui-angular-themes/assets/133086327/f7614f44-313e-408c-bad7-36359fd33f99)


<!-- Instruction for PR reviewers, if more complicated than a simple yarn start -->

#### To Test:

- yarn test

<!-- Useful for draft pull requests -->

#### Any specific feedback you are looking for?

-

JeffGreiner-eaton commented:
@RaghavendranC-Eaton what branch does this go in?

daileytj reviewed (commented):
Can you add a Changelog entry as well as a minor package version update to the package.json

RaghavendranC-Eaton commented:
> Can you add a Changelog entry as well as a minor package version update to the package.json

Added the issue fixed detail in the Changelog and updated the package version in the package.json
